utils/utils.go

```diff
@@ -401,7 +401,18 @@ func ReplaceOrAppendEnvValues(defaults, overrides []string) []string {
 		parts := strings.SplitN(e, "=", 2)
 		cache[parts[0]] = i
 	}
+
 	for _, value := range overrides {
+		// Values w/o = means they want this env to be removed/absent
+		// Note that this assumes an env var can never have a : in it
```

Comment:
I struggle to parse this comment and understand what it means.

Can we consider using `::docker::env::remove` or something even less likely to appear in the environment than `::remove`, in other words, to namespace it? Also, `:` is perfectly valid as an environment value.